presto-native-execution/presto_cpp/main/types/PrestoToVeloxPlanValidator.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+#include "velox/core/PlanFragment.h"
+
+namespace facebook::presto {
+class PrestoToVeloxPlanValidator {
+ public:
+  virtual bool validatePlanFragment(const velox::core::PlanFragment& fragment);
```

Comment:
perhaps either leave as pure virtual, or leave the base implementation here in the header so you don't need to define a .cpp that doesn't really have anything?